storage-rdma/src/main/java/com/ibm/crail/storage/rdma/client/RdmaStorageLocalEndpoint.java

```diff
@@ -136,23 +119,19 @@ public StorageFuture read(CrailBuffer buffer, BlockInfo remoteMr, long remoteOff
 			throw new IOException("remote offset too small " + remoteOffset);
 		}
 		
-		CrailBuffer mappedBuffer = bufferMap.get(remoteMr.getLkey());
+		long alignedLba = (remoteMr.getLba() / RdmaConstants.STORAGE_RDMA_ALLOCATION_SIZE) * RdmaConstants.STORAGE_RDMA_ALLOCATION_SIZE;
+		long lbaOffset = remoteMr.getLba() % RdmaConstants.STORAGE_RDMA_ALLOCATION_SIZE;
```

Comment:
Can we put these two lines to get the offset and align the lba in a method. Also used above.
Same goes for the checks at the beginning of the method same for both read and write.